pkg/db/queries.sql

```diff
@@ -46,13 +46,15 @@ DELETE FROM staged_originator_envelopes
 WHERE id = @id;
 
 -- name: SelectVectorClock :many
-SELECT
+SELECT DISTINCT ON (originator_node_id)
```

Comment:
can you help me understand this change?

Reply:
We are now returning the last envelope as well, so using GROUP BY is not going to work. What we want is one row per originator_node_id, not just the largest sequence ID value.

There's a good section on `DISTINCT ON` in https://www.postgresql.org/docs/current/sql-select.html if that helps!